WordPress/src/main/java/org/wordpress/android/ui/posts/PreviewStateHelper.kt

```diff
@@ -55,7 +55,7 @@ class PreviewStateHelper @Inject constructor() {
                         PostInfoType.PostNoInfo
                 )
                 if (!UploadService.isPostUploadingOrQueued(post)) {
-                    UploadService.uploadPost(activity, post, true)
```

Comment:
This change is a bit misleading - I removed `boolean isRemoteAutosave` parameter and I replaced it with `boolean isFirstTimePublish` parameter. Whether the post will be remote-auto-saved or auto-uploaded is decided in the `PostUploadHandler`.